packages/paste-website/src/pages/components/combobox/index.mdx

```diff
@@ -104,67 +106,96 @@ export const pageQuery = graphql`
 The Combobox is an opinionated implementation and wrapper around the **very excellent [Downshift component](https://downshift.netlify.app/downshift)**.
 
 Combobox allows a user to make a selection from a styled list box of options. Each option can consist of more than just text,
-_e.g._ text paired with an icon. It can also be set up with autocomplete/typeahead functionality so users can easily find a
+_e.g._ text paired with an icon. It can also be set up with autocomplete functionality so users can easily find a
 specific option.
```

Comment:
Change to this from the spec:

`A Combobox allows a user to select a single value from a styled listbox of options. Each option can consist of more than just text, _e.g._ text paired with an icon. However, an option can only be paired with non-interactive content and shouldn’t contain components like a checkbox or button. It can also be set up with autocomplete functionality so users can easily find a specific option.`